src/test/unit_tests/test_bucket_replication.js

```diff
@@ -920,4 +920,30 @@ mocha.describe('Replication pagination test', function() {
         // Make sure all 11 objects were replicated
         await list_objects_and_wait(s3_owner, target_bucket, obj_amount);
     });
+});
+
+mocha.describe('AWS S3 server log parsing tests', function() {
+    // Pagination test
+    mocha.it('Test AWS S3 server log parsing', async function() {
+        const logs = [];
+        const example_log = {Body: `
+        aaa test.bucket [13/Feb/2023:15:08:28 +0000] 1.1.1.1 arn:aws:iam::111:user/user AAA BATCH.DELETE.OBJECT test - 1 - - 1 - - - - - AAA SigV4 ECDHE-RSA-AES128-GCM-SHA256 AuthHeader s3.us-east-2.amazonaws.com TLSv1.2 - -
```

Comment:
I think we can also test for the following (as part of this PR or maybe future PRs):
1. If timestamp of a DELETE op on key `k` is greater than timestamp of a `PUT` op on key `k` **but** in the logs the `DELETE` shows up first - do we make sure that we process `DELETE` and not overwrite it with `PUT`? (Sagi found this)